pkg/controller/application/rolloutblock.go

```diff
@@ -0,0 +1,52 @@
+package application
+
+import (
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	apputil "github.com/bookingcom/shipper/pkg/util/application"
+	rolloutBlockOverride "github.com/bookingcom/shipper/pkg/util/rolloutblock"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Conventionally, imports are split into newline-padded sections like: stdlib, externals, internals. I believe k8s one might be moved above into external dependencies section and be padded from shipper imports.